pkg/kubelet/kubelet_node_status.go

```diff
@@ -474,6 +475,24 @@ func (kl *Kubelet) recordNodeSchedulableEvent(node *v1.Node) error {
 // TODO(madhusudancs): Simplify the logic for setting node conditions and
 // refactor the node status condition code out to a different file.
 func (kl *Kubelet) setNodeStatus(node *v1.Node) {
+	// TODO(dixudx): remove this in v1.14
+	// OutOfDisk condition has been removed from kubelet in 1.12
+	// For legacy node conditions, this should be removed when upgrading
+	if version.Get().Major == "1" && version.Get().Minor > "12" {
```

Comment:
Why do we need a version guard here?  We should only be cherry-picking back to 1.13, so this should always evaluate to true.

Reply:
agree, no version check necessary

Reply:
@dashpole @liggitt Addressed. PTAL. Thanks.